tests/reporter.test.ts

```diff
@@ -27,7 +30,7 @@ describe('default reporter', () => {
       },
       { logger },
     )
-    expect(logger).toHaveBeenCalledTimes(1)
+    expect(logger).toHaveBeenCalledTimes(2)
```

Comment:
My brain is not currently at full capacity– why did the call count need to increase here?

Reply:
That makes sense, could we actually assert the messages the logger is called with here (and below) for a bit of extra clarity?

Reply:
Done, you actually already asserted the firt call, I "copied" your assertion model for the second call.